public/index.html

```diff
@@ -3,24 +3,28 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
   </head>
-  <body>
-    <h1>Hello from Hoodie!</h1>
 
-    <p>You see this because your app has no <code>public/</code> folder.</p>
-    <p>But don't worry, we're here to get you started! </p>
+    <body>
+      <h1>Well done, you made it!</h1>
+    <div id="doggy"><img src="low-profile-dog-1.png" /></div>
```

Comment:
Ok so if I understand correctly the root element is `<html>`and since `<body>` is its children it should get one identation.